examples/plot_toy_int_experiment.py

```diff
@@ -143,7 +144,10 @@
 # MDS of tunning matrices
 corr_tunning = compute_pearson_corr(tuning_pred_run_1, tuning_pred_run_2)
 T_first_part_transformed, T_second_part_transformed = matrix_MDS(
-    tuning_pred_run_1, tuning_pred_run_2, n_components=2, dissimilarity=1 - corr_tunning
+    tuning_pred_run_1,
+    tuning_pred_run_2,
+    n_components=2,
+    dissimilarity=1 - corr_tunning,
```

Comment:
Will ruff complain if this is formatted as 
```suggestion
    dissimilarity=(1 - corr_tunning),
```
? I find it more human-readable, but not worth breaking automation !

Reply:
Both `ruff format` and `ruff check` don't seem to complain so I'm adding it.